app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/dashboard/DashboardScreen.kt

```diff
@@ -28,14 +32,21 @@ import com.twain.interprep.presentation.ui.components.PastInterviewCard
 import com.twain.interprep.presentation.ui.components.UpcomingInterviewCard
 import com.twain.interprep.presentation.ui.modules.interview.QuotesViewModel
 import com.twain.interprep.data.ui.QuoteData
+import com.twain.interprep.presentation.ui.components.InterviewCard
+import com.twain.interprep.presentation.ui.components.InterviewCardColor
+import com.twain.interprep.presentation.ui.modules.interview.InterviewViewModel
 
 @Composable
 fun DashboardScreen(
     navController: NavHostController,
-    viewModel: DashboardViewModel = hiltViewModel(),
+    viewModel: InterviewViewModel = hiltViewModel(),
     quotesViewModel: QuotesViewModel = hiltViewModel()
 ) {
+    // TODO ask Arighna
     quotesViewModel.insertQuotes(QuoteData.quotes)
+    LaunchedEffect(Unit){
```

Comment:
Do we require LaunchedEffect ?

LaunchedEffect: run suspend functions in the scope of a composable

To call suspend functions safely from inside a composable, use the [LaunchedEffect](https://developer.android.com/reference/kotlin/androidx/compose/runtime/package-summary#LaunchedEffect(kotlin.Any,kotlin.coroutines.SuspendFunction1)) composable. When LaunchedEffect enters the Composition, it launches a coroutine with the block of code passed as a parameter. The coroutine will be cancelled if LaunchedEffect leaves the composition.

Reply:
Although it does not harm to use LaunchedEffect for one time job. So ignore my previous comment.